packages/select/src/ResourceSelect.js

```diff
@@ -92,6 +104,19 @@ const ResourceSelect = ({
         ...rest.parameters,
       };
     }
+
+    if (typeof params === 'function') {
```

Comment:
Same here, we should be checking if `rest.parameters` is a function and not params